pytorch_lightning/callbacks/quantization.py

```diff
@@ -82,6 +83,64 @@ def _recursive_hasattr(obj: Any, attribs: str, state: bool = True) -> bool:
 
 
 class QuantizationAwareTraining(Callback):
+    r"""
+    Quantization allows speeding up inference and decreasing memory requirements
+    by performing computations and storing tensors at lower bitwidths
+    (such as INT8 or FLOAT16) than floating point precision.
+    We use native PyTorch API so for more information
+    see `Quantization <https://pytorch.org/docs/stable/quantization.html#quantization-aware-training>`_.
+
+    .. warning:: ``QuantizationAwareTraining`` is in beta and subject to change.
+
+    The model set for quantization can appear in one of this stages::
+
+        TRAINER TRANSITIONS
+                          ( on_fit_start )               ( on_fit_end )
+        MODEL STATES
+            vanilla model      --->       QuantAwareTrain     --->     quantized model
+                               /                |
+                    Trainer --/                 |
+            ( resume_from_checkpoint )          v
+                       ^                  QAT checkpoints
+                       |----------------------/
+
+    The model enters the process as "vanilla model" and it is prepared for QAT training in ``on_fit_start`` hook.
+    Note that any saved checkpoint includes already collected stat fro performing Quantization conversion,
+     but not any already quantized and/or fused modules/layers.
+    The quantization is performed on the ``on_fit_end`` and so it need sto he saved extra after finished training.
+    If a user wants to continue any past training we encourage to create a Trainer with ``resume_from_checkpoint``.
```

Comment:
```suggestion
    The model enters the process as "vanilla model" and it is prepared for QAT training in the ``on_fit_start`` hook.
    Note that any saved checkpoint includes already collected stats for performing Quantization conversion,
    but not any already quantized and/or fused modules/layers.
    The quantization is performed in the ``on_fit_end`` hook and so it needs to be saved extra after training finished.
    If a user wants to continue any past training we encourage to create a Trainer with ``resume_from_checkpoint``.
```

@Borda what do you mean by "so it needs to be saved extra"?